src/_incydr_cli/cmds/watchlists.py

```diff
@@ -312,21 +313,21 @@ def add(
 
     # Add included users
     if users:
-        client.watchlists.v1.add_included_users(
-            watchlist, _get_user_ids(client, users, format_=format_)
-        )
-        console.print(
-            f"Successfully included users on watchlist with ID: '{watchlist}'"
-        )
-
-    # Add excluded users
+        user_ids = _get_user_ids(client, users, format_=format_)
+        for chunk in chunked(user_ids, size=100):
+            client.watchlists.v1.add_included_users(watchlist, chunk)
+            console.print(
+                f"Successfully included {len(chunk)} users on watchlist with ID: '{watchlist}'"
+            )
+
+        # Add excluded users
     if excluded_users:
-        client.watchlists.v1.add_excluded_users(
-            watchlist, _get_user_ids(client, excluded_users, format_=format_)
-        )
-        console.print(
-            f"Successfully excluded users from watchlist with ID: '{watchlist}'"
-        )
+        user_ids = _get_user_ids(client, excluded_users, format_=format_)
+        for chunk in chunked(user_ids, size=100):
```

Comment:
What happens if one of these fails? 

Do we want to specify the behavior on failure and that passing more than 100 users will result in multiple requests?

Reply:
I added some more robust error handling, now we should proceed through any 'not found' errors and process the rest successfully. 

I also called out in the help text that the CLI will batch if > 100 users.